tests/it/dom-snapshot.edge.it.test.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const {describe, it, before, after} = require('mocha');
+const {expect} = require('chai');
+const {resolve} = require('path');
+const createTestCafe = require('testcafe');
+const testPath = resolve(__dirname, 'testcafe-tests/*.testcafe.js');
+const testServer = require('../util/testServer');
+
+describe('Eyes integration tests', () => {
+  let closeTestServer;
+  before(async () => {
+    const server = await testServer({port: 7272});
+    closeTestServer = server.close;
+  });
+
+  after(async () => {
+    await closeTestServer();
+  });
+
+  it('works', async () => {
+    let testcafe = null;
+    let failedCount;
+    await createTestCafe('localhost', 1337)
+      .then(tc => {
+        testcafe = tc;
+        const runner = testcafe.createRunner();
+        return runner
+          .src([testPath])
+          .browsers(['saucelabs:MicrosoftEdge@18.17763:Windows 10'])
```

Comment:
The test works when run by itself, but not as part of a suite. Seems like something odd in the `sauce-connect-launcher`